imports/plugins/core/accounts/client/containers/addressBookContainer.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { compose } from "recompose";
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { i18next } from "/client/api";
+import * as Collections from "/lib/collections";
+import { registerComponent, composeWithTracker } from "@reactioncommerce/reaction-components";
+import AddressBook from "../components/addressBook";
+
+/**
+ * @method updateAddress
+ * @summary helper function that updates an address in the account's addressBook via a meteor method.
+ * @since 2.0.0
+ * @param {Object} address - address to be updated.
+ * @param{String} property - property to be updated.
+ */
+function updateAddress(address, property) {
+  if (property) {
+    Meteor.call("accounts/addressBookUpdate", address, null, property);
+  } else {
+    Meteor.call("accounts/addressBookUpdate", address);
+  }
+}
```

Comment:
You should deal with any potential errors by having `Meteor.call` callbacks like the other two functions have. But also, I recommend making all of these return promises, which reject in case of error. Then, rather than calling `Alerts.toast` directly in these action functions, you should supply an `onError` function to all of the components.

So for example removeAddress becomes:

```js
function removeAddress(_id) {
  return new Promise((resolve, reject) => {
    Meteor.call("accounts/addressBookRemove", _id, (error, result) => {
      if (error || !result) {
        reject(i18next.t("addressBookGrid.cantRemoveThisAddress", { err: error && error.message }));
      } else {
        resolve(result);
      }
    });
  });
}
```

And then in your component you do

```js
onRemove = (_id) => {
  const { onError, removeAddress } = this.props;
  removeAddress(_id).catch(onError);
}
```

passing an `onError` similar to:

```js
function onError(errorMessage) {
  Alerts.toast(errorMessage, "error");
}
```

This keeps better separation which will make testing and debugging easier.